src/pages/Pokemon/Info.js

```diff
@@ -1,8 +1,9 @@
 import React from "react";
 import "./pokemon.css";
 
+
 const Info = ({ pokemonItem }) => {
-  const test = async () => {
+  async () => {
```

Comment:
mate sorry but i reckon we should move all the api logic to the parent component - Pokemon.js, and then pass the props down into this component to keep it dumb